tests/AnnotationScannerTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+use Collective\Annotations\AnnotationScanner;
+use Collective\Annotations\Events\Annotations\Scanner;
+use Doctrine\Common\Annotations\AnnotationRegistry;
+use Illuminate\Container\Container;
+use Illuminate\Support\Facades\App;
+use PHPUnit\Framework\TestCase;
+use Mockery as m;
+
+class AnnotationScannerTest extends TestCase
+{
+    /**
+     * @var m\LegacyMockInterface|m\MockInterface|Illuminate\Contracts\Foundation\Application
+     */
+    protected $app;
+
+    public function setUp(): void
+    {
+        $this->app = m::mock('Illuminate\Contracts\Foundation\Application');
+    }
+
+    public function tearDown(): void
+    {
+        m::close();
+    }
+
+    /**
+     * @covers \Collective\Annotations\AnnotationScanner::addAnnotationNamespace
+     * @covers \Collective\Annotations\AnnotationScanner::registerAnnotationsPathWithRegistry
+     * @covers \Collective\Annotations\NamespaceToPathConverterTrait::getPathFromNamespace
+     */
+    public function testAddAnnotationNamespaceWithoutSpecifiedPath()
+    {
+        $this->app->shouldReceive('getNamespace')->once()
+            ->andReturn('App\\');
+        Container::setInstance($this->app);
+
+        App::shouldReceive('make')
+            ->with('path')->once()
+            ->andReturn('src');
+
+        $scanner = new Scanner(['App\Handlers\Events\BasicEventHandler']);
+
+        $return = $scanner->addAnnotationNamespace(
+            'App\Events\Annotations\Annotations'
```

Comment:
Would it make sense to use `Annotations::class` here over `App\Events\Annotations\Annotations`?

Reply:
No as this isn't actually a class. It's a namespace that also doesn't technically exist, but due to how the test is set up, this namespace is mapped to the `Events/Annotations/Annotations` dir which we use to actually test the scanner. 
I'll add some more comments to the test, as it is a bit confusing